Dockerfile

```
@@ -0,0 +1,32 @@
+FROM node:10
```

```suggestion
FROM node:14
```
Update node image version to 14